adapters/hibernate/src/test/java/org/atomhopper/hibernate/HibernateFeedRepositoryTest.java

```diff
@@ -39,11 +41,8 @@
 @RunWith(Enclosed.class)
 public class HibernateFeedRepositoryTest {
 
-    public static class WhenPerformingSimpleAction {
```

Comment:
Why are you removing the WhenPerformingSimpleAction and the WhenPerformingComplexAction test classes?  

Should WhenCreatingMisconfiguredFeedRepository be an additional class rather than replacing?